appendix-basic65.tex

```diff
@@ -5838,8 +5833,10 @@ \subsection{NEXT}
 \item [Token:] \$82
 \item [Format:] {\bf FOR} index {\bf =} start {\bf TO} end
 		[{\bf STEP} step] ... {\bf NEXT} [index]
-\item [Usage:] Terminates the definition
-               of a BASIC loop with an index variable.
+\item [Usage:] Marks the end of the BASIC loop with the associated with the given index variable. When
```

Comment:
Remove the first `with the` (there's two presently.

So should be: `Marks the end of the BASIC loop associated with the given index variable.`

Reply:
done